renderer/components/Settings/SettingsFieldsWallet.js

```diff
@@ -30,7 +31,17 @@ const blockExplorerItems = [
   { key: 'smartbit', value: 'Smartbit' },
 ]
 
-const SettingsFieldsWallet = ({ currentConfig }) => {
+const SettingsFieldsWallet = ({ currentConfig, intl }) => {
+  const invoicesExpireItems = [
```

Comment:
since this is quite a big map, could be a good idea to move it one level above to avoid re-calculation on every render. I know that we use this style in other places. Their maps are usually smaller (2-3 items) but probably we shouldn't place them inside mappers as well.

Reply:
It's here because it relies on `intl` so it can't be generated statically outside of the render.

Reply:
I've extracted this out into a helper method and assed the intl module to it. It doesn't avoid recalculation on each render but it is a little cleaner.